test/e2e/policies.test-suite/topology-aware/n4c16/test11-reserved-cpu-annotations/code.var.sh

```diff
@@ -0,0 +1,29 @@
+# Test that
+# - containers marked in Annotations pinned on Reserved CPUs.
+
+cri_resmgr_cfg_orig=$cri_resmgr_cfg
+
+cleanup-test-pods() {
+    ( kubectl delete pods pod0 --now ) || true
+    ( kubectl delete pods pod1 --now ) || true
+}
+cleanup-test-pods
+
+terminate cri-resmgr
+AVAILABLE_CPU="cpuset:8-11"
+RESERVED_CPU="cpuset:10-11"
+cri_resmgr_cfg=$(instantiate cri-resmgr-reserved-annotations.cfg)
+launch cri-resmgr
+
+ANNOTATIONS='prefer-reserved-cpus.cri-resource-manager.intel.com/pod: "true"'
+CONTCOUNT=1 create reserved-annotated
+report allowed
+
+ANNOTATIONS='prefer-reserved-cpus.cri-resource-manager.intel.com/container.special: "false"'
+CONTCOUNT=1 create reserved-annotated
+report allowed
+
+verify 'cpus["pod0c0"] == {"cpu10", "cpu11"}'
+verify 'cpus["pod1c0"] == {"cpu08"}'
+
+cleanup-test-pods
```

Comment:
I'd suggest relaunching cri-resmgr with the original configuration as part of the clean up after successful test run. This would enable the following tests to start from similar state as this test started from without explicitly forcing cri-resmgr restart.

Reply:
Sure, np